prow/autobump-config/test-infra-autobump-config.yaml

```diff
@@ -12,9 +14,9 @@ includedConfigPaths:
   - "."
```

Comment:
Please replace dot with directory path or at least place some comment about what this dot represent.

Reply:
It represents what it's supposed to represent - the current directory the tool is working on. This will be the root directory of test-infra. This is self-explanatory

Absolute path has no sense since it can change depending on environment it is working on.

Reply:
But it doesn't say which directory it is. How often this directory changes in our environment? How I can know which is a current directory the tool is working on? Do I need to dig in to the code instead having clear information what it is? You don't have to use absolute path, a comment that this is a root directory of test infra would be enough. You did work on this tool. You spent time with it and you know everything. But others who might check this config later could have less knowledge.

Reply:
Added comment.